src/types/model/activities/call_activity.ts

```diff
@@ -0,0 +1,53 @@
+import {BpmnType} from '../../constants';
+import {Activity} from './activity';
+
+/**
+ * Describes a BPMN call activity.
```

Comment:
I am not sure if this writing is correct, either way you are doing it like this below so we should stay consistent.

```suggestion
 * Describes a BPMN CallActivity.
```